webpack.config.dev.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
     path: path.resolve(__dirname, './build'),
     pathinfo: true,
     filename: 'app/js/[name].bundle.js',
-    publicPath: '/',
+    publicPath: './',
```

Comment:
Any idea why this change is needed now when it was working before? or was it never working?

Reply:
would the client error we were seeing in INT PRs about public path because of this change?

Reply:
it is not because of this change.. this change is because of those errors. However, I'm going to undo this one because it should work since it worked before

Reply:
reverted that change